### DISCUSSION TITLE
训练数据集的问题

### DISCUSSION THREAD

**Discussion by Eason-shu** [PaddlePaddle/PaddleOCR#13599] [Q&A] [answered]
加入少量真实数据（检测任务>=500张, 识别任务>=5000张），会大幅提升垂类场景的检测与识别效果 ,这个是指我们的图片数据还是用PPOCRLabelv2标注过后的crop_img中的数据集数据？

**Comment by GreatV** [comment, marked as answer]
检测是实际图片的数据，识别是crop后的图片数量。

**Reply by Eason-shu** [reply]
好的，感谢
